firstLayer/converter.go

```diff
@@ -43,14 +33,16 @@ func UseFirstLayerSettings(argv []string) error {
 		}
 		return nil
 	})
-	if err != nil {
-		return err
+
+	if err != nil && err != gcode.ErrEarlyExit {
```

Comment:
Actually I may have been wrong about this; it looks like `gcode.ReadByLine` itself handles `gcode.ErrEarlyExit` and returns `nil` for us already. Worth double-checking but I think this can be reverted; sorry about that

Reply:
ahh yes, thanks for noting this. I missed it as well 😅